src/OpenLoco/Map/TileLoop.hpp

```diff
@@ -28,4 +28,82 @@ namespace OpenLoco::Map
         }
     };
 #pragma pack(pop)
+
+    struct TilePosLoop
```

Comment:
should it be called `TilePosLoopIterator`?

Reply:
I'm not sure iterator would be the correct name. Iterator pretty much only means the sub class of this. I was perhaps thinking TilePosRange or TilePosRangeView

Reply:
either of those work for me, I like the *View one as it describes what this class does accurately